src/request/create/Create.css

```diff
@@ -1,63 +1,74 @@
-.page .actionbox p {
-    max-width: 32rem;
-    margin-left: auto;
-    margin-right: auto;
-}
-
+.page#set-passphrase .page-body,
 .page#choose-identicon .page-body {
     position: relative;
+    display:flex;
+    flex-direction: column;
+    align-items: center;
 }
 
-.page#choose-identicon .page-body p:last-of-type {
-    margin-bottom: 2rem;
+.page#set-passphrase .page-body h2,
+.page#choose-identicon .page-body h2 {
+    text-align: center;
+    font-weight: normal;
 }
 
-.page-footer {
-    padding: 1rem;
+.page#set-passphrase.repeat-password .hide-on-repetition,
+.page#set-passphrase:not(.repeat-password) .show-on-repetition  {
+    display: none;
 }
 
-.keyfile-info {
-    display: flex;
-    flex-direction: row;
+.page-footer {
+    padding: 1rem;
 }
 
-.keyfile-info p:first-child {
-    margin-top: 0;
+.page-header {
+    padding-top: 5.5rem;
 }
 
-.keyfile-info p:last-child {
-    margin-bottom: 0;
+.page-header .progress-indicator {
+    position: absolute;
+    left: 0;
+    top: 0;
 }
 
-.nq-icon.walletfile {
-    width: 9rem;
-    height: 11.75rem;
-    margin-right: 4rem;
-    flex-shrink: 0;
+.page-header .page-header-back-button {
+    top: 5.5rem;
 }
 
-.page#recovery-words .skip-container {
+.wallet-identifier {
+    width: 20rem;
+    height: 20rem;
     text-align: center;
 }
 
-.page#recovery-words .skip {
-    color: inherit;
-    cursor: pointer;
+.wallet-identifier .identicon {
+    width: 7rem;
+    height: 6rem;
+    position: absolute;
+    animation: pop-in-0-to-115 0.4s cubic-bezier(.73,2.08,.48,1);
+    animation-delay: 300ms;
+    animation-fill-mode: both;
 }
 
-.page-header {
-    padding-top: 5.5rem;
+@keyframes pop-in-0-to-115 {
+    0% { transform: translate(-0.5rem, 2.5rem) scale(0); }
+    100% { transform: translate(0.2rem, 3.5rem) scale(1.15); }
 }
 
-.page-header .progress-indicator {
+.wallet-identifier .blank {
+    width: 7rem;
+    height: 6rem;
     position: absolute;
-    left: 0;
-    top: 0;
+    background-image: url("data:image/svg+xml,<svg width='58' height='49' viewBox='0 0 58 49' fill='none' xmlns='http://www.w3.org/2000/svg'><path opacity='0.08' d='M56.8082 22.2407L44.9478 2.67489C44.1017 1.27518 42.5283 0.417297 40.8362 0.417297H17.1154C15.4233 0.417297 13.8578 1.28271 13.0117 2.67489L1.14344 22.2407C0.297401 23.6404 0.297401 25.3562 1.14344 26.7559L13.0038 46.3217C13.8499 47.7214 15.4233 48.5792 17.1154 48.5792H40.8362C42.5283 48.5792 44.0938 47.7138 44.9399 46.3217L56.8002 26.7559C57.6542 25.3637 57.6542 23.6404 56.8082 22.2407Z' fill='%231F2348'/></svg>");
```

Comment:
In the dataurl above, you are using single quotes for the string, and double quotes for the svg attributes. This is the other way around. I like the first approach better.

Reply:
I agree.